lib/labels/ec2/ec2.go

```diff
@@ -109,25 +109,42 @@ func (l *EC2) Sync(ctx context.Context) error {
 
 	tags, err := l.c.Client.GetTagKeys(ctx)
 	if err != nil {
+		if trace.IsNotFound(err) {
```

Comment:
I assume this works because AWS is returning HTTP 404 and the trace package is able to convert it to a `NotFoundError`? Did you run this on a real instance to be sure that this works?